### DISCUSSION TITLE
Offline/online sync transaction issue

### DISCUSSION THREAD

**Discussion by LalitChauhan3093** [apache/cordova#476] [General]
Hi, I am facing a problem with cordova-ios 7.x.
Could you please advise me?

My app work online/offline - application works with Punch In punch out. If network available then all punch transaction sync to server otherwise it hold the transaction in offline mode and once network available then transaction sync automatically. 

For transaction holding in offline I am using plugin sqlitePlugin its working fine till April ends but Now, we start facing the issue i.e., Its hold the transaction and unable to sync the records.

I update all the plugins such as(Sqlite, file, file transfer etc).

If is there any one have any idea please advice me.

Thank you

**Comment by breautek** [comment]
Your question is pretty open-ended, for example:

> we start facing the issue i.e., Its hold the transaction and unable to sync the records.

Could literally mean anything and doesn't help isolate what the issue could be, or where the issue is. Therefore the first thing I'd recommend is determine if it's a client side issue or a server side issue, and finding an actual error message or error code that would indicate what the issue might be.
